### ISSUE TITLE
Ensure we are using personal sign

### ISSUE THREAD

**Issue by pi0neerpat** [oneclickdapp/ethereum-auth#4]
Community members for Swordy Bot have pointed out that using eth_sign is not a good behavior to encourage with users.

> In particular, the method eth_sign is an open-ended signing method that allows signing an arbitrary hash, which means it can be used to sign transactions, or any other data, making it a dangerous phishing risk.

> For this reason, we make this method show the most frightening possible message to the user, and generally discourage using this method in production. However, some applications (usually admin panels internal to teams) use this method for the sake of its ease of use, and so we have continued to support it for the sake of not breaking the workflows of active projects.

TL;DR Every time we use `eth_sign`, we are teaching people its ok to sign things that could be bad!

I don't know what Meta Mask's plan is here, but based on this, I would recommend they add back the scary warning message they used to have. Otherwise developers will continue to use `eth_sign`.

![image](https://user-images.githubusercontent.com/35622595/138190151-b29aa95a-f60d-479d-8e11-a157b7eea5c8.png)

**Comment by jyap808** [comment]
Updating to a newer https://www.npmjs.com/package/@ethersproject/providers might resolve this. There’s been some relevant fixes.

Possibly related: https://github.com/ethers-io/ethers.js/commit/8947fd405e3aea07f6db958d89a3ad39abe3a25a
